src-web/tableDefinitions/utils.js

```diff
@@ -402,9 +402,12 @@ export function formatAnnotationString(policy, annotationKey){
 }
 
 export function getAutomationLink(item, locale) {
-  return (
-    <AutomationButton item={item} locale={locale}></AutomationButton>
-  )
+  if (_.get(item, 'metadata.namespace')) {
```

Comment:
Is this check necessary? I can’t think of the case where namespace would be empty.

Reply:
just in case the passed-in `item` don't have `metadata.namespace` then the whole page will blank，now the worst  case is showing `-` on ansible filed